tests/kernel/common/prj.conf

```diff
@@ -6,3 +6,5 @@ CONFIG_BOOT_DELAY=500
 CONFIG_IRQ_OFFLOAD=y
 CONFIG_TEST_USERSPACE=y
 CONFIG_BOUNDS_CHECK_BYPASS_MITIGATION=y
+CONFIG_TIMESLICING=y
```

Comment:
isnt this enabled by default?

Reply:
I check this in the zephyr doc and it is by default. Later, I'll check this doc when enable new configurations. Thanks for reminding.

Reply:
I've removed this configuration in code.